### PR TITLE
Feat/publish-gate-qa

### PR DESCRIPTION
**Pull request by NunoMartins21**
# [PR Name]

## Type of PR

- [x] - feature
- [x] - bugfix
- [x] - chore

## Description

Fixed some bugs mainly related to the Publish Gate flow, but also some other minor things that I found across the website

## Checklist

- [x] - Provided useful description for this PR
- [] - Provided the root cause of this bugfix
- [x] - Tested on supported devices

### CODE REVIEW

**Review by fbenfraj — APPROVED** [review]
Lot of cool changes! 😄 LGTM ✅

File: apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx [hunk 1]
```diff
@@ -21,26 +22,36 @@ export function UserCell({ user }: Props) {
...
-              variant="caption"
-              color="text.secondary"
-              sx={{
-                textOverflow: 'ellipsis',
-                overflow: 'hidden',
-                whiteSpace: 'nowrap',
-              }}
+      <TableCell
+        sx={{
+          cursor: 'pointer',
```
Comment (kbooz) [inline apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx:27]: don't need cursor pointer if stack is an anchor

File: apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx [hunk 2]
```diff
@@ -21,26 +22,36 @@ export function UserCell({ user }: Props) {
...
+        }}
+      >
+        <Link href={'/profile/' + user.username} passHref>
+          <Stack alignItems="center" direction="row" gap={1}>
```
Comment (kbooz) [inline apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx:31]: put `<Stack component="a"`

**Review by kbooz — CHANGES_REQUESTED** [review]

File: apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx [hunk 3]
```diff
@@ -26,27 +28,29 @@ export function UserCell({ user }: Props) {
...
-                overflow: 'hidden',
-                whiteSpace: 'nowrap',
-              }}
-            >
-              @{user.username}
-            </Typography>
-          </Box>
-        </Stack>
+        <Link href={'/profile/' + user.username} passHref>
+          <Stack alignItems="center" direction="row" gap={1}>
```
Comment (kbooz) [inline apps/website/components/templates/dao-profile/tabs/people-tab/user-cell.tsx:32]: add `component="a"`

**Review by kbooz — CHANGES_REQUESTED** [review]

**Pull request merged**